src/axom/primal/operators/detail/intersect_impl.hpp

```diff
@@ -1209,6 +1209,490 @@ AXOM_HOST_DEVICE bool intersect_plane_seg(const Plane<T, 3>& plane,
   return false;
```

Comment:
@bmhan12 -- can you please confirm that you used `git mv` to preserve the history on this file?

Reply:
> @bmhan12 -- can you please confirm that you used `git mv` to preserve the history on this file?

No, I essentially appended the contents of `intersect_impl.cpp` to the end of `intersect_impl.hpp`. 

Is there a way to use `git mv` to preserve that migration and history of `intersect_impl.cpp`?